todo_app/tests/test_integration.py

```diff
@@ -0,0 +1,157 @@
+import pytest, os
+from dotenv import find_dotenv, load_dotenv
+import todo_app.app as app
+from unittest.mock import patch, Mock
+
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    test_app = app.create_app()
+    with test_app.test_client() as client: 
+        yield client
+
+@patch('requests.get')
+def test_index_page(mock_get, client):
+    mock_get.side_effect = mock_get_cards
+    response = client.get('/')
+    print(response.data.decode())
+    assert response.status_code == 200
+
+def mock_get_cards(url, params=None):
+
+    if url == f'https://trello.com/1/lists/todo_list_id/cards?key=test_key&token=test_token': 
+        response = Mock()
+        response.json.return_value = sample_trello_cards_response()
+        return response
+
+    if url == f'https://trello.com/1/lists/done_list_id/cards?key=test_key&token=test_token': 
+        response = Mock()
+        response.json.return_value = sample_trello_cards_response()
+        return response
+    
+    return None
+
+def sample_trello_cards_response():
+    return [
+    {
+        "id": "60969ef0795a5e4c0e063550",
+        "checkItemStates": null,
```

Comment:
This is written as if it's raw JSON but is actually a python dictionary - so you'll see errors here due to e.g. use of `null` rather than `None`, or `false` instead of `False`. We should either adjust them here, or leave this as a string that contains a JSON object that we could read in

Reply:
Done, thanks.